Packs/CommonScripts/Scripts/TextFromHTML/TextFromHTML.py

```diff
@@ -5,26 +5,59 @@
 import re
 
 
-def get_plain_text(html_regex):
+def get_plain_text(html: str, replace_line_breaks: bool, trim_result: bool):
     data = ''
-    if html_regex and html_regex.group(0):
-        data = re.sub(r'<.*?>', '', html_regex.group(0))
+    if html:
+        data = re.sub(r'<\/?br\s?\/?>', '\n', html, flags=re.I) if replace_line_breaks else html
+
+        data = re.sub(r'<.*?>', '', data)
         entities = {'quot': '"', 'amp': '&', 'apos': "'", 'lt': '<', 'gt': '>', 'nbsp': ' ',
                     'copy': '(C)', 'reg': '(R)', 'tilde': '~', 'ldquo': '"', 'rdquo': '"', 'hellip': '...'}
         for e in entities:
             data = data.replace(f'&{e};', entities[e])
+
+        if trim_result:
+            data = re.sub(r'[ \t]{2,}', ' ', data)
```

Comment:
```suggestion
            data = re.sub(r'\s{2,}', ' ', data)
```

Reply:
Making this change would result in line 21 being useless, as \s also matches Linebreaks.
The Ideas of Line 20 is: Remove occurences of two or more consecutive space or tab characters from the extracted text, whereas line 21 intends to avoid large blocks of "empty" lines.
changing the regex to \s, would also match strings like "some line\n\nwith a empty line between", which isn't intended.
I would be happy to merge them together, but for line 21, having 3 or more is intentional, to allow for single empty lines, whereas for tabs or space, in most cases, having two or more is already something i wanted to avoid.